index.html

```diff
@@ -290,6 +290,18 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    
+    <div class="footer">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/mvodvrnv" method="post" class="form">
+        <input type="text" placeholder="Full name" name="name" required>
+        <input type="email" placeholder="Email address" name="email" required>
+        <textarea name="message" id="text" placeholder="Enter your text" required></textarea>
```

Comment:
- [ ] Same here, the message input should have a `maxlength="500"` attribute. This is to prevent unnecessarily long messages.